packages/layout/src/components/cards/components/button.module.scss

```diff
@@ -65,6 +67,21 @@
 	width: 100%;
 }
 
+.removeButton {
+	button:focus {
+		background-color: yellow;
```

Comment:
It would make sense for this `yellow` value to be added to `packages/theming/src/variables.scss` as something like `$colors-focus-background` and then to use the variable here.

Reply:
I was awaiting confirmation from Jodi that this was the right color. I need to make few changes.